city_scrapers/spiders/tulok_citycouncil.py

```
@@ -0,0 +1,361 @@
+
+class TulsaGranicusCityCouncilSpider(CityScrapersSpider):
+    name = "tulok_citycouncil"
+    agency = "Tulsa City Council"
+    timezone = "America/Chicago"
+    start_urls: ClassVar[list[str]] = [
+        "https://tulsa-ok.granicus.com/ViewPublisher.php?view_id=4"
+    ]
+    location = {
+        "name": "City Hall",
+        "address": "175 E 2nd St, Tulsa, OK 74103",
+    }
```

Ruff (0.14.8)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)